samples/bluetooth/direction_finding_connectionless_rx/README.rst

```diff
@@ -0,0 +1,162 @@
+.. _direction_finding_connectionless_rx:
+
+Bluetooth: Direction finding connectionless locator
+###################################################
+
```

Comment:
Move 1 intro sentence from the Overview section to here, e.g. the first one: The direction finding connectionless locator sample application demonstrates Bluetooth LE direction finding reception.

Reply:
Moved :-)